api/tests/unit/sse/test_sse_service.py

```diff
@@ -96,3 +103,63 @@ def test_send_environment_update_message_for_environment_schedules_task_correctl
             realtime_enabled_project_environment_one.updated_at.isoformat(),
         )
     )
+
+
+@mock_s3
+def test_stream_access_logs(mocker: MockerFixture, settings):
+    # Given - Some test data
+    first_log = SSEAccessLogs("2023-11-27T06:42:47+0000", "key_one")
+    second_log = SSEAccessLogs("2023-11-27T06:42:47+0000", "key_two")
+    third_log = SSEAccessLogs("2023-11-27T06:42:47+0000", "key_three")
+
+    first_encrypted_object_data = b"first_bucket_encrypted_data"
+    first_decrypted_object_data = (
+        f"{first_log.generated_at},{first_log.api_key}\n"
+        f"{second_log.generated_at},{second_log.api_key}".encode()
+    )
+    second_encrypted_object_data = b"second_bucket_encrypted_data"
+    second_decrypted_object_data = (
+        f"{third_log.generated_at},{third_log.api_key}".encode()
+    )
+
+    # patch the s3 resource because it was created before the mock_s3 decorator was applied
+    patch_resource(s3)
```

Comment:
I don't 100% follow what patch_resource does. Does it re-enable S3 for the test?